webapp/test/App.spec.js

```diff
@@ -1,24 +1,56 @@
 import { mount } from '@vue/test-utils';
 import app from '../src/App.vue';
+import {USER, AUTH_TOKEN} from '../src/constants/settings.js'
 
-describe('App.vue', () => {
+const title = "Frontend-Einkaufslisten-Bearbeitungs-Und-Erstellungsmaschine";
+const notLoggedInInfo = "You are not logged in.";
+const loginFormDivName = "loginForm";
+const todoListDivName = "todoList";
 
+describe('App.vue', () => {
   let wrapper = mount(app);
-  beforeEach(() => {
-    wrapper = mount(app);
-  });
 
-  it('renders the todo app', () =>{
-    expect(wrapper.contains('div')).toBe(true);
-  });
+  describe("When App is loaded", () => {
+    beforeEach(() => {
+      wrapper = mount(app);
+    });
+
+    it("renders the login form", () => {
+      const loginForm = wrapper.find({name: loginFormDivName});
+      expect(loginForm.is("div")).toBe(true);
+    });
+    it("shows title & loginInfo", () => {
+      expect(wrapper.text()).toContain(title);
+      expect(wrapper.text()).toContain(notLoggedInInfo)
+    });
+    it('does not show todo list', () =>{
+      expect(wrapper.contains({name: todoListDivName})).toBe(false)
+    });
+
+    it('has no local storage attributes', () =>{
+      expect(localStorage.getItem(USER)).toBeNull();
+      expect(localStorage.getItem(AUTH_TOKEN)).toBeNull;
+    });
 
-  describe('when the toggleEditModeb function is called', () => {
-    //TODO reimplement with backend mockup
-    //Test fails after apollo backend integration
-    // it('opens an input field in a listitem component', () => {
-    //     const editButton = wrapper.findAll('button').at(2);
-    //     editButton.trigger('click');
-    //     expect(wrapper.html()).toContain('input');
-    // });
+    // dont test logging in because its not testing only components?
```

Comment:
Yes, if you want to test the registration flow with emails, this is certainly out-of-scope of a component test. You would need a fullstack test for this. Honestly, I would not test emails because it's too complex and expensive for no real benefit.